src/components/KCollapse/KCollapse.vue

```diff
@@ -104,21 +110,21 @@ export default defineComponent({
       default: '',
     },
     triggerAlignment: {
-      type: String,
+      type: String as PropType<TriggerAlignment>,
       required: false,
-      default: 'trailing',
-      validator: (value: string): boolean => {
-        return ['leading', 'trailing'].includes(value)
+      default: triggerAlignmentRecord.trailing,
```

Comment:
When we move to script setup we won't be able to utilize variables as the prop defaults like this; I think you just need to hard-code it.